app/middlewars/authMiddleware.js

```diff
@@ -0,0 +1,15 @@
+import config from '../config';
+import jwt from 'jsonwebtoken';
+
+export const authMiddleware = (req, res, next) => {
+    if (req.path === '/login') return next();
```

Comment:
it's better to use this middleware after public routes middlewares

Reply:
done